.vscode/launch.json

```diff
@@ -137,7 +137,7 @@
       "request": "launch",
       "preLaunchTask": "prepare turbo",
       "program": "${workspaceRoot}/target/debug/turbo",
-      "args": ["daemon"],
+      "args": ["--skip-infer", "daemon"],
```

Comment:
What's the reason for this change?

Reply:
It's for debugging, and it ensures that we run the daemon that we're intending to debug, rather than whatever the repo depends on at the time.